free-2.md

```diff
@@ -57,13 +70,15 @@ spent directly engaging in the **language** aspects of this project (e.g.,
 making language design decisions), as opposed to "systems" aspects of the
 project (e.g., implementing a complicated semantics that doesn't require a lot
 of language design)?_
+I think that it would be largely language design because a programmer would have to decide exactly what programs would look like, how they would be implemented and design both sides. But there would be a significant amount of systems programming to execute the communication between the two sides.
 
 
 ### Scope
 _How big an idea is this? How ambitious is this project?_
+I think this is a pretty big idea because of the multiple interfaces. I think its pretty ambitious but achievable.
 
 
 ### Benefits and drawbacks
 _Why might this be a good idea for a project? Why might this not be a good idea 
 project?_
-
+This might be a good idea because I think that, especially at schools like mudd, it would be solving a very real problem and save paper! It might not be a good idea for a project because it might have more application building aspects and less language design stuff (maybe).
```

Comment:
This may be a case where the domain is too small to warrant a language.
I don't see this language being _expressive_, though I may just have a misunderstanding of what the project is.